packages/ubuntu_bootstrap/lib/services/installer_service.dart

```diff
@@ -33,16 +36,28 @@ class InstallerService {
 
   Future<void> load() async {
     await monitorStatus().firstWhere((s) => s?.isLoading == false);
-    _pages ??=
-        await _client.getInteractiveSections() ?? await _config?.getPages();
+    _subiquityPages = (await _client.getInteractiveSections())?.toSet();
+    _excludedPages = _pageConfig?.excludedPages;
+
+    if ((_subiquityPages?.isNotEmpty ?? false) &&
+        (_excludedPages?.isNotEmpty ?? false)) {
+      final requiredExcludedPages =
+          _excludedPages!.intersection(_subiquityPages!);
+      _log.info(
+          '$requiredExcludedPages are required by subiquity and cannot be excluded!');
+      _excludedPages!.removeAll(requiredExcludedPages);
+    }
```

Comment:
```suggestion
    final requiredExcludedPages =
          _excludedPages?.intersection(_subiquityPages ?? {});
    if (requiredExcludedPages.isNotEmpty) {
      _log.info(
          '$requiredExcludedPages are required by subiquity and cannot be excluded!');
      _excludedPages!.removeAll(requiredExcludedPages);
    }
```
Super nitpicking :sweat_smile:
Just improves readability a bit.